### ISSUE TITLE
Will the interleaved data opensource?

### ISSUE THREAD

**Issue by lucasjinreal** [LLaVA-VL/LLaVA-NeXT#70]
Will the interleaved data opensource?

**Comment by FengLi-ust** [comment]
Yes, the training data is under preparation. We will release it soon.

**Comment by lucasjinreal** [comment]
Thats very good, how does the quality compares with Cambricon?

**Comment by FengLi-ust** [comment]
Can you give a link for the work Cambricon?

**Comment by syspider** [comment]
> Yes, the training data is under preparation. We will release it soon.

quite excited

**Comment by zihaolucky** [comment]
Hi @FengLi-ust is https://huggingface.co/datasets/lmms-lab/M4-Instruct-Data the right one?

**Comment by FengLi-ust** [comment]
Yes, we have just released the data.

**Comment by xiaoachen98** [comment]
> Yes, we have just released the data.

Thanks for your excellent work. What about the video data part? Will this part of the data be released?
